openstacksdk_plugin/exception.py

```diff
@@ -0,0 +1,18 @@
+# #######
+# Copyright (c) 2018 Cloudify Platform Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+#    * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    * See the License for the specific language governing permissions and
+#    * limitations under the License.
+
+
+class UnSupportedOperation(Exception):
```

Comment:
Nice, but `UnsupportedOperation` would be more correct. Still it's is a little generic, what do you think about `CloudifyOpenstackPluginError`?

Why isn't this a `NonRecoverableError` and if so, why not just use `NonRecoverableError`? Otherwise, why isn't it a `RecoverableError`?

Reply:
I think you are right, this should be `NonRecoverableError ` not `UnsupportedOperation`